src/guides/appchain-register.md

```diff
@@ -1,8 +1,62 @@
 ## Appchain Register
 
+### Checklist before registration
+
+This section aims to list the items that the Appchain team needs to check before Appchain registration to ensure a smooth launch.
+
++ The Appchain is using the latest version of the barnacle/barnacle-evm template.
++ The version of Substrate used in Cargo.toml of Appchain should be consistent with the template.
++ The below pallets defined in the `construct_runtime!` part in the Appchain runtime code (runtime/src/lib.rs) should be consistent with the template.
+
+```rust
+construct_runtime!(
+	pub enum Runtime where
+		Block = Block,
+		NodeBlock = opaque::Block,
+		UncheckedExtrinsic = UncheckedExtrinsic
+	{
+		System: frame_system,
+		Babe: pallet_babe,
+		Timestamp: pallet_timestamp,
+		Authorship: pallet_authorship,
+		Balances: pallet_balances,
+		TransactionPayment: pallet_transaction_payment,
+		OctopusAppchain: pallet_octopus_Appchain, // must before session
+		OctopusLpos: pallet_octopus_lpos,
+		OctopusUpwardMessages: pallet_octopus_upward_messages,
+		OctopusAssets: pallet_assets::<Instance1>,
+		Session: pallet_session,
+		Grandpa: pallet_grandpa,
+		ImOnline: pallet_im_online,
+		Historical: pallet_session_historical::{Pallet},
+		Mmr: pallet_mmr,
+		Beefy: pallet_beefy,
+		MmrLeaf: pallet_beefy_mmr,
+		Uniques: pallet_uniques,
+		Sudo: pallet_sudo,
+		// Include the custom logic from the pallet-template in the runtime.
+		TemplateModule: pallet_template,
+	}
+);
+```
+
++ The block time of the Appchain is consistent with the template, set to 6s.
+
+```rust
+pub const MILLISECS_PER_BLOCK: Moment = 6000;
+pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 4 * HOURS;
+```
++ In Appchain chain spec, ss58Format is 42, if you do not need to transfer the stablecoin assets (USDN) of the main chain (NEAR network) to Appchain, please set `assetIdByTokenId` with `[]`.
```

Comment:
In Appchain chain spec JSON file, the setting of `ss58Format` should be 42. Also, if Appchain does not support the stablecoin assets (USDN in the NEAR network) cross-chain transfer from the Mainchain to Appchain, please set `assetIdByTokenId` with `[]`.